datafusion/sqllogictest/test_files/joins.slt

```diff
@@ -1336,6 +1350,29 @@ physical_plan
 10)----------RepartitionExec: partitioning=RoundRobinBatch(2), input_partitions=1
 11)------------MemoryExec: partitions=1, partition_sizes=[1]
 
+
+statement ok
+set datafusion.optimizer.prefer_hash_join = true;
```

Comment:
this has to be by default, did we change the value before the test?

Reply:
Removed, left over from me trying to understand why it did not use HashJoin